server/subscribe.go

```diff
@@ -124,7 +125,7 @@ func (p *Plugin) getUserID() string {
 	return p.getConfig().botUserID
 }
 
-func (p *Plugin) matchesSubsciptionFilters(wh *webhook, filters SubscriptionFilters) bool {
+func (p *Plugin) matchesSubsciptionFilters(wh *webhook, filters SubscriptionFilters, visibilityAttribute string) bool {
```

Comment:
I'm thinking we can add a `commentVisibility` field onto the `webhook` struct, and avoid adding this `visibilityAttribute` param to multiple functions